Initial Nunit tests reworked and added back to solution with…

tsd-sageorgiev opened the pull request:
Initial Nunit tests reworked and added back to solution with new logic dependencies.

All test are passing (for integration test servername, username and password must be entered)
VSUnit test removed

tsd-sageorgiev committed:
Initial Nunit tests reworked and added back to solution with new logic dependencies.
All test are passing (for integration test servername, username and password must be entered)
VSUnit test removed

ppande1020 commented on Source/TextExtractor.Agents.NUnit/ManagerJobTests.cs:
```diff
@@ -0,0 +1,70 @@
+		}
+
+		[Category(TestCategory.UNIT)]
+		[Test]
```
Can we please add description for every test? something like this 
[Test(Description = "Verifies that the application exits if a user cancels the job between the time it was entered and the agent picking it up ")]

tsd-sageorgiev replied:
Sure, I'll add description for every test

ppande1020 reviewed (approved):
As discussed approving this request so we can see the rest of the tests that Svet has worked on. Thanks - Priya

tsd-sageorgiev committed:
Include/Exclude marker to extracted text choice.
Tests for new functionality in EventHandlers, Helpers.
Description added to all Tests.
Removed commented out GUIDs for kCura.Talos.Utility.

tsd-sageorgiev committed:
Check for Source Text Field length not to exceed 4 million characters
